packages/quick_actions/quick_actions_ios/example/ios/Podfile

```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-# platform :ios, '9.0'
+# platform :ios, '11.0'
```

Comment:
these changes are automatically added when i do flutter build ios

Reply:
You need to manually undo them; it's a change that's on `master` but hasn't rolled out to `stable`, and we keep our plugin OS support the same as Flutter stable's OS support (except in cases where we can't for technical reasons).